lib/Parser/String.pm

```diff
@@ -17,11 +17,12 @@ sub new {
   my $equation = shift; my $strings = $equation->{context}{strings};
   my ($value, $ref) = @_;
   my $def = $strings->{$value};
+  my $VALUE = uc($value);
```

Comment:
Cool, so `uc` not only changes the contents of the variable to upper case, it also changes the variables name to upper case!  I like your naming scheme.

Reply:
:-)

This commit still has one problem that I am trying to fix.  But have had to stop for the moment.  I'll let you know when it is correct.